src/gsy_e/gsy_e_core/live_events.py

```diff
@@ -57,7 +57,7 @@ def apply(self, area):
             self.created_area.parent = area
             area.children.append(self.created_area)
             if self.created_area.strategy:
-                self.created_area.strategy.event_activate()
+                self.created_area.strategy.activate()
```

Comment:
I see, the SCM base class method is just activate() contrary to event_activate(). Makes sense